lib/services/simple/access_monitoring_rules.go

```diff
@@ -0,0 +1,97 @@
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package simple
+
+import (
+	"context"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/api/types/accessmonitoringrule"
+	"github.com/gravitational/teleport/lib/backend"
+	"github.com/gravitational/teleport/lib/services"
+	"github.com/gravitational/teleport/lib/services/local/generic"
+	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	accessMonitoringRulePrefix      = "access_monitoring_rule"
+	accessMonitoringRuleMaxPageSize = 100
+)
+
+// AccessMonitoringRuleService is a simple access monitoring rules backend service for use specifically by the cache.
```

Comment:
This service seems basically identical to the one defined in `lib/service/local`, except that there are no calls to `CheckAndSetDefaults` here. Is that omission worth creating an other service just for the cache?